google/cloud/storage/tests/object_integration_test.cc

```diff
@@ -93,6 +93,23 @@ non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                         "abcdefghijklmnopqrstuvwxyz012456789");
   }
 
+  void GenerateRandomLines(std::ostream& upload, std::ostream& local) {
```

Comment:
Nit: Given a function named `GenerateRandomLines`, I'd expect it to return a collection of random lines.  Maybe name this `WriteRandomLines` or `WriteRandomLinesToStreams`?

Reply:
Done.